README.md

```diff
@@ -13,33 +13,24 @@ between both the server and the client) for SEO reasons.
 [![Dependency Status](https://david-dm.org/quran/quran.com-frontend.svg)](https://david-dm.org/quran/quran.com-frontend) [![devDependency Status](https://david-dm.org/quran/quran.com-frontend/dev-status.svg)](https://david-dm.org/quran/quran.com-frontend#info=devDependencies)
 [![Code Climate](https://codeclimate.com/github/quran/quran.com-frontend.png)](https://codeclimate.com/github/quran/quran.com-frontend)
 
-## Getting started
-Simply clone this repo, then run `npm install` to install all the required node_modules.
-From there, you are ready to go!
+## How to contribute
+We trust that
+you will not steal this, this is at the end of the day for the sake of Allah and we
```

Comment:
I think we should not worry based on few assumptions:
1 - A common person can't understand all this technology mess, so his idea of theft will not be successful 
2 - Only a professional can understand this project and professionals prefer to craft their own Art.

May ALLAH protect this project from all thieves and hackers. Ameen